.circleci/config.yml

```diff
@@ -1,59 +1,74 @@
-version: 2.1
+version: 2.1  # Version of CircleCI config format
 
-orbs:
-  codecov: codecov/codecov@1.1.0
+orbs:  # CircleCI packages ("orbs") we depend on
+  codecov: codecov/codecov@1.1.0  # support for uploading code coverage to codecov
 
 executors:
   my-executor:
     docker:
       - image: humancompatibleai/seals:base
+        # Credentials defined in `docker-hub-creds` context.
+        # We authenticate to avoid Dockerhub per-IP rate limiting of anonymous users.
         auth:
           username: $DOCKERHUB_USERNAME
           password: $DOCKERHUB_PASSWORD
-    working_directory: /benchmark-environments
+    working_directory: /seals
     environment:
       # If you change these, also change ci/code_checks.sh
       SRC_FILES: src/ tests/ setup.py
       TYPECHECK_FILES: src/  # tests often do weird things with types, exclude
```

Comment:
The current version of this comment doesn't feel like it fully gives me clarity about why `src/` is in this category, probably because I'm not very familiar with what `TYPECHECK_FILES` signifies